watertap/unit_models/electrodialysis_0D.py

```diff
@@ -1546,6 +1717,184 @@ def eq_dl_thickness_ioa(self, mem, side, t):
                     * self.current_dens_lim_ioa[t] ** -1
                 )
 
+    def _get_fluid_dimensionless_quantities(self):
+        self.diffus_mass = Var(
+            initialize=1e-9,
+            bounds=(1e-16, 1e-6),
+            units=pyunits.meter**2 * pyunits.second**-1,
+            doc="The mass diffusivity of the solute as molecules (not individual ions)",
+        )
+        self.hydraulic_diameter = Var(
+            initialize=1e-3,
+            bounds=(0, None),
+            units=pyunits.meter,
+            doc="The hydraulic diameter of the channel",
+        )
+        self.N_Re = Var(
+            initialize=50,
+            bounds=(0, None),
+            units=pyunits.dimensionless,
+            doc="Reynolds Number",
+        )
+        self.N_Sc = Var(
+            initialize=2000,
+            bounds=(0, None),
+            units=pyunits.dimensionless,
+            doc="Schmidt Number",
+        )
+        self.N_Sh = Var(
+            initialize=100,
+            bounds=(0, None),
+            units=pyunits.dimensionless,
+            doc="Sherwood Number",
+        )
+
+        if self.config.hydraulic_diameter_method == HydraulicDiameterMethod.fixed:
+            _log.warning("Do not forget to FIX the channel hydraulic diameter in [m]!")
+        else:
+
+            @self.Constraint(
+                doc="To calculate hydraulic diameter",
+            )
+            def eq_hydraulic_diameter(self):
+                if (
+                    self.config.hydraulic_diameter_method
+                    == HydraulicDiameterMethod.conventional
+                ):
+                    return (
+                        self.hydraulic_diameter
+                        == 2
+                        * self.channel_height
+                        * self.cell_width
+                        * self.spacer_porosity
+                        * (self.channel_height + self.cell_width) ** -1
+                    )
+                else:
+                    self.spacer_specific_area = Var(
```

Comment:
This should also have bounds and doc for consistency

Reply:
Bounds and docs were added for both 0D and 1D files.